src/aat/java/uk/gov/hmcts/reform/em/annotation/functional/AnnotationSetScenarios.java

```diff
@@ -1,9 +1,12 @@
 package uk.gov.hmcts.reform.em.annotation.functional;
 
```

Comment:
One more scenario with a HTTP Status Forbidden (403) would be very good...

Reply:
This api doesn't have role based access, so 403 is not required.

Reply:
Yes I agree this can't be tested immediately...